docs/docker-options.md

```diff
@@ -0,0 +1,60 @@
+docker-options
+========================
+
+Usage
+-----
+
+```bash
+$ dokku help
+...
+    docker-options <app>                               Display apps docker options for all phases
+    docker-options <app> <phase(s)>                    Display apps docker options for phase (comma seperated phase list)
+    docker-options:add <app> <phase(s)> OPTION         Add docker option to app for phase (comma seperated phase list)
+    docker-options:remove <app> <phase(s)> OPTION      Remove docker option from app for phase (comma seperated phase list)
+...
+````
+
+Add some options
+
+```bash
+$ dokku docker-options:add myapp deploy "-v /host/path:/container/path"
+$ dokku docker-options:add myapp run "-v /another/container/path"
+$ dokku docker-options:add myapp "-link container_name:alias"
+```
+
+Check what we added
+
+```bash
+$ dokku docker-options myapp
+-link container_name:alias
+-v /host/path:/container/path
+-v /another/container/path
+```
+
+Remove an option
+```bash
+$ dokku docker-options:remove myapp "-link container_name:alias"
+```
+
+Manual Usage
+------------
```

Comment:
Lets make this `Advanced Usage` and have a note to users that they should avoid this workflow if at all possible.